.github/workflows/deploy.yml

```diff
@@ -0,0 +1,44 @@
+name: Deploy
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sendgrid-csharp
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Might not need to do this as we don't need an extensive git history here because we don't build a docker image